test/linter/test-browsers-data.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+const path = require('path');
+const chalk = require('chalk');
+const { Logger } = require('./utils.js');
+
+/**
+ * @typedef {import('../../types').Identifier} Identifier
+ * @typedef {import('../utils').Logger} Logger
+ */
+
+/**
+ * @param {Identifier} data
+ * @param {Logger} logger
+ * @returns {boolean}
+ */
+function processData(data, logger) {
+  const browser = Object.keys(data.browsers)[0];
+  const releases = data.browsers[browser].releases;
+
+  let releaseByStatus = {
+    current: null,
+    beta: null,
+    nightly: null,
+    esr: null,
+  };
+
+  for (let releaseVersion in releases) {
+    const releaseData = releases[releaseVersion];
+
+    if (['current', 'beta', 'nightly', 'esr'].includes(releaseData.status)) {
+      if (
+        releaseData.status === 'esr' &&
+        ['nodejs', 'firefox'].includes(browser)
```

Comment:
Can we even ban the `esr` value from all other browsers? It seems there are two cases here: multiple ESR releases or no ESR at all.

Reply:
To be honest, NodeJS and Firefox are really the only two that _have_ ESR versions to begin with.  I guess we don't really need to add the special-case logic because of that.